coduno.go

```diff
@@ -13,73 +13,138 @@ import (
 
 	"github.com/coduno/app/controllers"
 	"github.com/coduno/app/mail"
+	"github.com/coduno/app/models"
 	"github.com/gorilla/mux"
+	"github.com/gorilla/sessions"
+	"github.com/m4rw3r/uuid"
 
 	"golang.org/x/net/context"
 
 	"google.golang.org/appengine"
+	"google.golang.org/appengine/datastore"
 	"google.golang.org/appengine/log"
 	"google.golang.org/appengine/urlfetch"
 )
 
 var gitlabToken = "YHQiqMx3qUfj8_FxpFe4"
+var sessionID = "SESSIONID"
+var store = sessions.NewCookieStore([]byte("c6fe0cba-5a74-478d-8389-a5d852c7fsae2b"))
 
 // Handler is similar to a HandlerFunc, but also gets passed
 // the current context.
 type Handler func(http.ResponseWriter, *http.Request, context.Context)
 
+// HandlerWithSession is similar to Handler, but it also receives a param that
```

Comment:
`func(http.ResponseWriter, *http.Request, context.Context)` does not receive a `bool`.